### ISSUE TITLE
Replace actions in the download dropdown

### ISSUE THREAD

**Issue by CGillen** [OregonDigital/OD2#3456]
### Descriptive summary
We would like to repurpose the `download` dropdown on the work show page.
We'll get rid of the metadata only option and add a new option for individual downloads which will scroll the user down to the "new" Works and FIles section. (#3455).
Accessibility remediated files should not be included with these download options and should only be available from the individual downloads in the `Works and Files` section.

### Expected behavior
`Download` dropdown only has the following options:
- Download files (High Quality)
- Download files (Low Quality)
- Download individual files

`Download files + metadata` and `metadata only` downloads are removed. This means we can remove all metadata downloading and packing logic from downloading.

Accessibility remediated files (Files marked true from the `schema:accessibilityFeature` field) are not included in downloadable zips.

### Related work
#3455
#3454

**Comment by lamtu1** [comment]
@CGillen I was able to get the download for all fileSets to work but I have questions about some of the detail:

1. When mention that **_Files marked true from the `schema:accessibilityFeature` field) are not included in downloadable zips_** If the FileSet have any value of accessibility Feature in it, don't include it in the file download? Can the only value 'unknown' within the field be downloadable?

2. When you mention this: _**Download individual files**_ Do you mean to download the default FileSet that get seen first? Because we have a download option for individual file within the Work & Files section in the `action` section

**Comment by CGillen** [comment]
They should be able to download it from the individual files section, yes

**Comment by lamtu1** [comment]
@KevinJonesMeta I will look and get the PR that has this to merge with v4 and get v4 updated with this code to test out on the new v4 version. I will have to look through to see any conflict it might have with v4 and try to get it up and running on the new staging as fast as possible.

**Comment by lamtu1** [comment]
hi @briesenberg07 Can you QA this in `staging`? We only got the 3 options on `prod` and it hasn't been push to production just yet. Can I have you QA in `staging` and update on what you see? Thank you!

**Comment by KevinJonesMeta** [comment]
@lamtu1 I'll look into that. I'll get some examples on prod for the complex object with high-quality download not logged in case and link here. (there has got to be an easier way to say that).
